Plugins/ActSVG/include/Acts/Plugins/ActSVG/SvgUtils.hpp

```diff
@@ -38,6 +38,7 @@ struct Style {
   std::vector<int> strokeDasharray = {};
 
   unsigned int fontSize = 14u;
+  std::array<int, 3> fontColor = {0, 0, 0};
```

Comment:
This should do the same thing, but up to you how explicit you want this:
```suggestion
  std::array<int, 3> fontColor = {0};
```